### PR TITLE
e2e test

### PR DESCRIPTION
**Pull request by jiyoontbd**

### CODE REVIEW

File: e2e/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    maven {
+      url = uri("https://jitpack.io")
+    }
+}
+
+dependencies {
+  implementation(project(":protocol"))
+  implementation(project(":httpclient"))
+  implementation("me.lessis:typeid:0.0.2")
+  implementation("com.github.TBD54566975:web5-kt:0.0.0-beta")
```
Comment (phoebe-lew) [inline e2e/build.gradle.kts:22]:
```suggestion
  implementation("com.github.TBD54566975:web5-kt:0.0.4-beta")
```

File: e2e/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  testImplementation(platform("org.junit:junit-bom:5.9.1"))
+  testImplementation("org.junit.jupiter:junit-jupiter")
+}
```
Comment (phoebe-lew) [inline e2e/build.gradle.kts:25]: I think we're also using `testImplementation(kotlin("test"))` in all the other packages

File: settings.gradle.kts [hunk 3]
```diff
@@ -1,2 +1,2 @@
 rootProject.name = "tbdex"
-include("httpclient", "httpserver", "protocol")
+include("httpclient", "httpserver", "protocol", "e2e")
```
Comment (phoebe-lew) [inline settings.gradle.kts:2]: Does this mean we'll publish `e2e` as a jitpack package @mistermoe ?

**Review by phoebe-lew — APPROVED** [review]

File: settings.gradle.kts (hunk 3 above)
Comment (jiyoontbd) [reply settings.gradle.kts:2]: i _think_ no. chad says https://chat.openai.com/share/d51bd87a-90fa-49c3-bb67-7b4a44b1b11b

**Pull request merged**